aws/assume-role/README.md

```diff
@@ -55,10 +55,57 @@ tasks:
     call: aws/install-cli 1.0.0
   - key: assume-role
     use: aws-cli
-    call: aws/assume-role 1.0.2
+    call: aws/assume-role 1.1.0
     with:
       oidc-token: ${{ vaults.your-vault.oidc.your-token }}
       region: us-east-2
       role-to-assume: arn:aws:iam::your-account-id:role/your-role
       profile-name: your-profile
 ```
+
+To assume another role, via chaining:
+
+```yaml
+tasks:
+  - key: aws-cli
+    call: aws/install-cli 1.0.0
+  - key: assume-role
+    use: aws-cli
+    call: aws/assume-role 1.1.0
+    with:
+      oidc-token: ${{ vaults.your-vault.oidc.your-token }}
+      region: us-east-2
+      role-to-assume: arn:aws:iam::your-account-id:role/your-role
+  - key: chained-role
+    use: assume-role
+    call: aws/assume-role 1.1.0
+    with:
+      chain: true
+      region: us-east-2
+      role-to-assume: arn:aws:iam::your-account-id:role/your-other-role
+```
+
+To assume another role, via chaining, with specific profiles:
+
+```yaml
+tasks:
+  - key: aws-cli
+    call: aws/install-cli 1.0.0
+  - key: assume-role
+    use: aws-cli
+    call: aws/assume-role 1.1.0
+    with:
+      oidc-token: ${{ vaults.your-vault.oidc.your-token }}
+      region: us-east-2
+      role-to-assume: arn:aws:iam::your-account-id:role/your-role
+      profile-name: your-profile
+  - key: chained-role
+    use: assume-role
+    call: aws/assume-role 1.1.0
+    with:
+      chain: true
```

Comment:
Do we need `chain: true`? Maybe it makes the usage more clear, but it seems like otherwise We could just require either `oidc-token` or `source-profile-name`.